packages/shared/src/components/cards/ActionButtons.tsx

```diff
@@ -146,25 +151,33 @@ export default function ActionButtons({
         {insaneMode && lastActions}
       </ConditionalWrapper>
       {insaneMode ? (
-        <div
-          className={classNames('flex justify-between', visibleOnGroupHover)}
-        >
-          {!isInternalReadType(post) && (
-            <ReadArticleButton
-              content={getReadPostButtonText(post)}
-              className="mr-2"
-              variant={ButtonVariant.Primary}
-              href={getReadArticleLink(post)}
-              onClick={onReadArticleClick}
-              openNewTab={!isSharedPostSquadPost(post) && openNewTab}
-            />
+        <>
+          {flagProps?.trending && (
+            <div className="absolute right-0 flex h-8 items-center rounded-10 bg-action-downvote-default px-3 py-1 font-bold text-white typo-callout laptop:mouse:group-hover:invisible">
+              Trending{' '}
+              <TrendingIcon className="ml-1" secondary size={IconSize.Small} />
+            </div>
```

Comment:
This seems to be the same to the one in packages/shared/src/components/cards/PostCardHeader.tsx, why not create a component?

Reply:
Slight difference and still experiment i guess .
Can you if you think its better for now?